web/cypress/integration/api/history.spec.js

```diff
@@ -0,0 +1,7 @@
+describe('History', () => {
```

Comment:
In the longer term, should this testing go into the app backend testing directly? Since here we are testing the output of something that we should trust (the app should assume that the API works and is returning things correctly).

Reply:
The idea is to have both types of testing. This should not be considered as our main API tests, it just verified that the app is able to query the backend correctly :)

Reply:
Is the way the app queries the data different from a regular API call? I'm trying to understand the difference and why this test is needed on top of the API tests.

Reply:
Yes, well the app queries the app-backend which again queries our standard API, so there's a middlestep which is the app-backend.

Ideally we want a bunch of unit tests for the app and the app-backend but now I am just testing the **integration** step, i.e. whether the app and the app-backend is able to communicate properly

So to put some words on these tests, it essentially tests that we're able to spin up the app and create authorized requests to the production-app-backend based on user actions, which may seem simple, but is a super nice sanity check for us before merging things

Reply:
Good to know, makes sense!